packages/cli/src/utils/files.js

```diff
@@ -200,9 +202,49 @@ const makeTempDir = () => {
   return workdir;
 };
 
+async function deleteUnmatchedFiles(src, dest, ig) {
```

Comment:
This function works! 

....until you have a gitignored directory (as you can see in the below TODO). So this'll need a refactor. My current thought is instead of taking a source and destination as params, these should be a source and destination list/array. Then they can be filtered, more closely matching the pattern we already have in `makeSourceZip` where we first list the files [then filter them](https://github.com/zapier/zapier-platform/blob/95680f62897064853fb147350d3a183b85509a2a/packages/cli/src/utils/build.js#L248) through gitignored and blacklisted.